cfpb/cfgov-refresh
# Add canonical URL tag to base template

Adds "canonical" metadata tag to base.html template in order to help inform search bots that our unique page content is important and unique. See https://GHE/CFGOV/platform/issues/2380

## Additions

- adds tag to the `<head>` of base.html template and therefore all the pages that use this template

## Removals

-

## Changes

-

## Testing

1. check out branch
1. run `./setup.sh`
1. run `./runserver.sh`
1. go to http://localhost:8000/ and click around various URLs and view source to confirm you see the tag and URL output in the `<head>` on line 25, e.g. `<link rel="canonical" href="http://localhost:8000/">`


## Screenshots
![screen shot 2018-01-17 at 1 35 04 pm](https://user-images.githubusercontent.com/702526/35060099-50021ffa-fb8b-11e7-8281-80105fd13430.png)


## Notes

-

## Todos

-

## Checklist
